src/components/Navigation.jsx

```diff
@@ -2,17 +2,35 @@ import React from 'react';
 import { Link } from 'react-router';
 
 export default class Navigation extends React.Component {
+
   render() {
+
+    // Render all categories in nav bar
+    let renderCategories = () => {
+      let categories = ["opinion", "features", "news"];
+      let data = [];
+
+      categories.forEach(function(category) {
+        data.push(
+          <li key={category}><Link to={"/category/"+category}>{category}</Link></li>
+        )
+      });
+
+      return data
+    }
+
     return (
       <div>
         <h1>{this.props.appName}</h1>
-        {/* Add list item by categories that exist in issue x */}
         <ul>
           <li><Link to="/">Home</Link></li>
-          <li><Link to="/issue/55/opinion/palestine-hamilton">Sample Article</Link></li>
-          <li><Link to="/author/ahmed-meshref">Sample Author</Link></li>
+          {renderCategories()}
```

Comment:
Again here, just assign the variable at top, no need to make it a function call.